### PR TITLE
Update MSTest element

### PR DESCRIPTION
**Pull request by engyebrahim**
Related to: https://github.com/microsoft/testfx/issues/1319

### CODE REVIEW

**Comment by Jak-MS** [comment]
@Mikejo5000 
- Can you review this PR?
- IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: docs/test/configure-unit-tests-by-using-a-dot-runsettings-file.md [hunk 1]
```diff
@@ -318,14 +318,18 @@ These settings are specific to the test adapter that runs test methods that have
...
-|**KeepExecutorAliveAfterLegacyRun**|false|After a test run is completed, MSTest is shut down. Any process that is launched as part of the test is also killed. If you want to keep the test executor alive, set the value to **true**. For example, you could use this setting to keep the browser running between coded UI tests.|
 |**DeploymentEnabled**|true|If you set the value to **false**, deployment items that you've specified in your test method aren't copied to the deployment directory.|
 |**CaptureTraceOutput**|true|You can write to the debug trace from your test method using <xref:System.Diagnostics.Trace.WriteLine%2A?displayProperty=nameWithType>.|
+|**EnableBaseClassTestMethodsFromOtherAssemblies**|true|a value indicating whether to enable discovery of test methods from base classes in a different assembly from the inheriting test class.|
```
Comment (Mikejo5000) [inline docs/test/configure-unit-tests-by-using-a-dot-runsettings-file.md:324]:
```suggestion
|**EnableBaseClassTestMethodsFromOtherAssemblies**|true|A value indicating whether to enable discovery of test methods from base classes in a different assembly from the inheriting test class.|
```

File: docs/test/configure-unit-tests-by-using-a-dot-runsettings-file.md [hunk 2]
```diff
@@ -318,14 +318,18 @@ These settings are specific to the test adapter that runs test methods that have
...
+|**ClassCleanupLifecycle**|EndOfClass|If you want the class cleanup to occer at the end of assimbly set it to **EndOfAssembly**.|
```
Comment (Mikejo5000) [inline docs/test/configure-unit-tests-by-using-a-dot-runsettings-file.md:325]:
```suggestion
|**ClassCleanupLifecycle**|EndOfClass|If you want the class cleanup to occur at the end of assembly, set this to **EndOfAssembly**.|
```

File: docs/test/configure-unit-tests-by-using-a-dot-runsettings-file.md [hunk 3]
```diff
@@ -318,14 +318,18 @@ These settings are specific to the test adapter that runs test methods that have
...
+|**MapNotRunnableToFailed**|true|A value indicating whether a not runnable result be mapped to failed test.|
```
Comment (Mikejo5000) [inline docs/test/configure-unit-tests-by-using-a-dot-runsettings-file.md:326]:
```suggestion
|**MapNotRunnableToFailed**|true|A value indicating whether a not runnable result will be mapped to failed test.|
```

File: docs/test/configure-unit-tests-by-using-a-dot-runsettings-file.md [hunk 4]
```diff
@@ -318,14 +318,18 @@ These settings are specific to the test adapter that runs test methods that have
...
+|**Parallelize**||You will need to use it to set the parallelization settings:<br /><br />**Workers**: the number of threads/workers to be used for parallelization and it is by default **the number of processors on the current machine**.<br /><br />**SCOPE**: the scope of parallelization, you can set it to **MethodLevel** and by default it's **ClassLevel**.<br /><br />`<Parallelize><Workers>32</Workers><Scope>MethodLevel</Scope></Parallelize>`|
```
Comment (Mikejo5000) [inline docs/test/configure-unit-tests-by-using-a-dot-runsettings-file.md:327]:
```suggestion
|**Parallelize**||You will need to use it to set the parallelization settings:<br /><br />**Workers**: The number of threads/workers to be used for parallelization, which is by default **the number of processors on the current machine**.<br /><br />**SCOPE**: The scope of parallelization. You can set it to **MethodLevel**. By default, it's **ClassLevel**.<br /><br />`<Parallelize><Workers>32</Workers><Scope>MethodLevel</Scope></Parallelize>`|
```

**Comment by Mikejo5000** [comment]
#sign-off @engyebrahim Thanks!

**Pull request merged**